languages/postman/src/main/kotlin/io/vrap/codegen/languages/postman/model/PostmanModuleRenderer.kt

```diff
@@ -113,7 +115,7 @@ class PostmanModuleRenderer constructor(val api: Api, override val vrapTypeProvi
             To use this collection in Postman please perform the following steps:
 
             1. Download and install the Postman Client
-            2. Import the [collection.json](https://github.com/commercetools/commercetools-postman-api-examples/raw/master/collection.json) and [template.json](https://github.com/commercetools/commercetools-postman-api-examples/raw/master/template.json) in your postman application
+            2. Import the [collection.json](${githubPath}/collection.json) and [template.json](${githubPath}/template.json) in your postman application
```

Comment:
```suggestion
            2. Import the [collection.json](collection.json) and [template.json](template.json) in your postman application
```

just use a relative path here ;)